janusgraph-dist/src/assembly/static/conf/remote-graph-binary.yaml

```diff
@@ -0,0 +1,17 @@
+# Copyright 2020 JanusGraph Authors
```

Comment:
Any reason not to simply change the serializer in `remote.yaml`? GraphBinary will replace Gryo in TinkerPop which is what we currently use in `remote.yaml`. I think that we should therefore also stop using that and make GraphBinary the new default.
Or do you want to wait a bit and then make this maybe the default in version 0.7.0?

Reply:
I think we shouldn't change this until we are able to manage a JanusGraph Server remotely without gryo.

Reply:
Good point, I didn't think about that use case.